shader/antialiasing/cmaa2/compute_dispatch_args.comp

```diff
@@ -0,0 +1,50 @@
+#version 460
+
+#extension GL_ARB_separate_shader_objects       : enable
+#extension GL_GOOGLE_include_directive          : enable
+#extension GL_EXT_control_flow_attributes       : enable
+
+#include "cmaa2_common.glsl"
+
+layout(binding = 8) buffer UboWorkingExecuteIndirectBuffer {
+  uint g_workingExecuteIndirectBuffer[];
+  };
+
+layout(local_size_x = 1, local_size_y = 1, local_size_z = 1) in;
+
+void main() {
+  uvec3 groupID = gl_WorkGroupID;
+
+  if(groupID.x == 1) {
```

Comment:
Here `gl_WorkGroupID` is uses as essentially a flag. Better to just use push-constant.